ses-ballerina/src/ses/tests/unit-tests/main_test.bal

```diff
@@ -0,0 +1,135 @@
+string emailVerificatioAddress = getConfigValue("VERIFY_ADDRESS");
+string emailSendFromAddress = getConfigValue("SEND_FROM_ADDRESS");
+string emailSendToAddress = getConfigValue("SEND_TO_ADDRESS");
+string templatedEmailSendFromAddress = getConfigValue("TEMPLATED_SEND_FROM_ADDRESS");
+string templatedEmailSendToAddress = getConfigValue("TEMPLATED_SEND_TO_ADDRESS");
+string templatedEmailSendCcAddress = getConfigValue("TEMPLATED_SEND_CC_ADDRESS");
+
+@test:Config {
+    groups: ["group1"]
+}
+function testVerifyEmailIdentity() {
+    error? response = sesClient->verifyEmailIdentity(emailVerificatioAddress);
+    if (response is error) {
+        log:printError("Error while trying to verify email address." + response.message());
+        test:assertTrue(false);
+    }
+}
+
+@test:Config {
+    dependsOn: ["testVerifyEmailIdentity"],
+    groups: ["group1"]
+}
+function testSendEmail() {
+    Email email = {
+        to: [emailSendToAddress],
+        subject: "Sample AWS SES Email Subject",
+        body: "Sample Email Body",
+        'from: emailSendFromAddress
+    };
+    error? response = sesClient->sendEmail(email);
+    if (response is error) {
+        log:printError("Error while sending an email." + response.message());
+        test:assertTrue(false);
+    }
+}
+
+@test:Config {
+    dependsOn: ["testVerifyEmailIdentity"],
+    groups: ["group2"]
+}
+function testCreateTemplate() {
+    Template template = {
+        templateName:templateName,
+        subjectPart: "Greetings, {{name}}!",
+        textPart: "Dear {{name}},\r\nYour favorite animal is {{favoriteanimal}}.",
+        htmlPart: "<h1>Hello {{name}}</h1><p>Your favorite animal is {{favoriteanimal}}.</p>"
+    };
+    error? response = sesClient->createTemplate(template);
+    if (response is error) {
+        log:printError("Error while trying to create a template." + response.message());
+        test:assertTrue(false);
+    }
+}
+
+@test:Config {
+    dependsOn: ["testCreateTemplate"],
+    groups: ["group2"]
+}
+function testSendTemplatedEmail() {
+    map<string> defaultTemplateData = {name:"friend", favoriteanimal:"unknown"};
+    BulkEmailDestination[] destinations = [
+        {
+            destination: {to:[templatedEmailSendToAddress]},
+            replacementTemplateData: {name:"Donald", favoriteanimal:"duck"}
+        },
+        {
+            destination: {cc:[templatedEmailSendCcAddress]},
+            replacementTemplateData: {name:"Obama", favoriteanimal:"eagle"}
+        }
+    ];
+    string[] replyToAddresses = [templatedEmailSendFromAddress];
+
+    EmailDestinationStatus[]|Error? response = sesClient->sendTemplatedEmail(templatedEmailSendFromAddress, templateName, destinations, defaultTemplateData,
+        replyToAddresses, templatedEmailSendFromAddress);
+    if (response is Error) {
+        log:printError("Error while trying to send a templated email." + response.message());
+        test:assertTrue(false);
```

Comment:
You can use `assertFail` function - https://ballerina.io/swan-lake/learn/api-docs/ballerina/test/functions.html#assertFail

Reply:
Changed.